lang/sql/index.md

```diff
@@ -39,14 +39,21 @@ AND should be dropped onto a newline and given an additional indentation of four
 IN, AS, and OR should all be placed on the same line as the clause they are a part of.
```

Comment:
These should be quoted as well, thought you'd see these too, but I should have called them out.